misk/src/main/kotlin/misk/metrics/Metrics.kt

```diff
@@ -0,0 +1,26 @@
+package misk.metrics
+
+import com.codahale.metrics.Counter
+import com.codahale.metrics.Gauge
+import com.codahale.metrics.Histogram
+import com.codahale.metrics.MetricRegistry
+import com.codahale.metrics.Timer
+import java.util.*
```

Comment:
Wildcard!

Reply:
argh. I finally found where it is automatically using wildcards for java.util.